fastlane/metadata/tr/subtitle.txt

```diff
@@ -1 +1 @@
-Bir web sitesi oluşturun
+Bir web sitesi veya blog oluşturun
```

Comment:
This will need to be reverted at some point because it's more than 30 characters.

I'm working on adding the 30 characters limit comment for the translators in the `.po` sources, which should help prevent this from happening again.